qml/DeviceManagerViewer.qml

```diff
@@ -148,6 +148,8 @@ PingPopup {
                             root.close()
                         }
                         ToolTip {
+                            // Position tooltip slightly above parent.
+                            y: parent.y - height - 5
```

Comment:
Use margin if possible, avoid magic numbers.

Reply:
I tried, bu couldn't get anything better to work.

Reply:
You can use the parent to use the mouse position and the height to make it more friendly and with a better position.
```
x: parent.mouseX
y: parent.mouseY - height - 5
```
![image](https://user-images.githubusercontent.com/1215497/62297146-12a52000-b447-11e9-9392-232f00d53965.png)
![deepin-screen-recorder_Select area_20190801102824](https://user-images.githubusercontent.com/1215497/62297181-25b7f000-b447-11e9-9d25-b84a082d2e14.gif)